### ISSUE TITLE
Homebridge still working on iPad but now "no response" on iPhone

### ISSUE THREAD

**Issue by RAGR1** [nfarina/homebridge#1494]
I have not changed anything but getting a "no response" for everything on my homebridge (Elk M1 and Nest accessories) on my iPhone. iPad continues to work beautifully. Any ideas where to start?
So far I've tried:
updating to current homebridge.
turning off iCloud keychain and restarting iPhone.
deleting and reinstalling Home app (I know that doesn't change anything but I thought I'd give it a try).

**Comment by zackaweed** [comment]
I suddenly have the exact same problem, everything was working fine last night. From the looks of the homebridge terminal everything is working normally server side, but all homebridge devices on the home app are "not responding". No idea what it may be.

**Comment by zemetfan** [comment]
Exactly the same for me :(
Everything went fine until last night : now I get "no response" for every accessories and no logs.

I have done the same steps as RAGR1 but no success.

**Comment by KhaosT** [comment]
What’s your iOS version?

**Comment by RAGR1** [comment]
iOS 10.3.3

**Comment by RAGR1** [comment]
Now iOS 11

**Comment by fahrvergnuugen** [comment]
This has been happening to me on and off for the past 1 year on various different versions of homebridge and iOS. My wife's iPhone will continue to work fine but mine will get the dreaded "No Response" message on all of my accessories. (I use HomeKit a lot more than she does)

I discovered that rebooting my phone will usually fix the problem, but sometimes I have to delete everything and start over (persist folder and re-add homebridge as a new accessory).

Do we know if this is a HomeBridge problem or a HAP-NodeJS problem?

**Comment by VeniceNerd** [comment]
Having the same issue as Fahrvergnuugen. I also noticed that if I make a backup of my “accessories” and “persist” folders when everything I working I can usually just put those back in after a fail to get the system going again. Seems like something in those folders gets messed up after a while.

**Comment by RAGR1** [comment]
Fixed issue: pulled the config.json out of the folder. deleted the accessories folder in ~homebridge folder. removed homebridge from home app on iphone. reinstalled homebridge (https://github.com/nfarina/homebridge). ran homebridge in terminal. stopped homebridge in terminal. put json file back into ~homebridge folder. ran homebridge again in terminal. added homebridge to home app. ------basically, just reinstalled everything. Working great.

**Comment by fahrvergnuugen** [comment]
@ragr1 until it happens again, which it most likely will do.

**Comment by fahrvergnuugen** [comment]
Small update since my last post.... @VeniceNerd has explained to me that he has both homebridge and other official homekit devices (like phillips hue) in his environment. He verified that this "no response" issue is occurring with homebridge accessories only, but not with his other officially licensed homekit accessories.

I also read another blog post where the author managed to fix his homebridge "no response" issue by adjusting the multicast settings on his router.

The fact that this can happen with homebridge only while other physical accessories continue to work rules out an issue with the router in my mind - multicast must be working or the other devices would not work.

This got me wondering if there is some problem or bug related to the MDNS npm package that homebridge relies on, but I haven't thought of a way to verify it.

Anyone have any thoughts on that?

**Comment by fahrvergnuugen** [comment]
I had a another experience which I think backs up my previous suspicion:

1. Accessories on iPhone were not responding while on wifi
1. I restarted the iPhone. All Homebridge accessories were still "not responding"
1. 20-30 minutes later without making any changes, all Homebridge accessories came back online

This leads me to believe that my phone simply could not reach or talk to homebridge. I need to find some iOS apps that can help me test my network & bonjour resolution for next time this occurs...

**Comment by fahrvergnuugen** [comment]
Another update: my **Not Responding** issue is 100% related to MDNS (Bonjour) not working correctly.

- Homebridge is running on my mac mini, named mini.local
- My iPhone shows "Not Responding" while my wife's iPhone works fine
- On my iPhone I have Bonjour Browser, Ping and VNC.
- I cannot see the mini with the VNC app which uses Bonjour for discovery
- I cannot ping the mini by mini.local, but I can ping by it's IP Address
- The output from Bonjour Browser between my wife's iPhone and mine is identical, so that is puzzling to me. It seems all services are discovered (I see Apple Remote Desktop, HAP, as well as my iTunes library on the mini, but it still doesn't show in VNC).
- After waiting approximately an hour, everything is working again

FWIW, My home router is a Netgear R6220 with firmware V1.1.0.34_1.0.1. The mini is connected on a wired port.

So it seems pretty cut and dry to me that this issue is related to MDNS or my local network, but I'm not quite sure where to go from here.

**Comment by DiMarcello77** [comment]
Hello All, I had this same issue and I solved it today. homebridge was found but all accesoires where 'Not Responding' 

I thought it was because of an IOS 11 update first but after completely removing / reinstalling my PI3 with homebridge the issue was still there. I googled a lot and in my case my router (Netgear Orbi) had this multicast feature set to **disabled**: 

_Disable IGMP Proxying. IGMP proxying allows a computer on the local area network (LAN) to receive the multicast traffic it is interested in from the Internet._

After enabling this multicast feature and rebooting the router it was completely running fine again!

It's unknown to me why this setting was disabled but now it is working like a charm again :-) 

Keep up the good work nfarina! I love homebridge!

**Comment by majestic1327** [comment]
Hello
I have homebridge installed for several month with absolutely no issue: it was working on the Apple TV, my two iPhones, my home is shared with my fiancee’s iPhone and everything was working fine until I brought back my iPad from my office. The iPad was off for nearly six month and as soon as I charge it, turn it on and unlock it, every other apple device stopped working with the red no response underneath each accessories.
All the device stopped working except the iPad.
I removed everything from the home app and removed the content of homebridge (content of persist folder and accessories folder), turn off the iPad and reinstall homebridge and bam! Everything worked.
As soon as I turned on the iPad everything stopped working.
I am thinking about selling my iPad now
